scripts/bsd/openbsd-prep-postgres.sh

```diff
@@ -28,6 +28,19 @@ pkg_add -I \
     openldap-client--gssapi \
     openldap-server--gssapi
 
+#####
+# Add 'noatime' and 'softdep' to the mount points
+# Taken from https://daemonforums.org/showthread.php?t=4378
+FSTAB_FILE=/etc/fstab
+LABELS=adefghijklm
```

Comment:
Uh, huh?

Reply:
These are labels of mount points, to use them in sed command, for example:

```bash
$ cat /etc/fstab
092510a73bfe1df6.b none swap sw
092510a73bfe1df6.a / ffs rw 1 1
092510a73bfe1df6.k /home ffs rw,nodev,nosuid 1 2
092510a73bfe1df6.d /tmp ffs rw,nodev,nosuid 1 2
092510a73bfe1df6.f /usr ffs rw,nodev 1 2
092510a73bfe1df6.g /usr/X11R6 ffs rw,nodev 1 2
092510a73bfe1df6.h /usr/local ffs rw,wxallowed,nodev 1 2
092510a73bfe1df6.j /usr/obj ffs rw,nodev,nosuid 1 2
092510a73bfe1df6.i /usr/src ffs rw,nodev,nosuid 1 2
092510a73bfe1df6.e /var ffs rw,nodev,nosuid 1 2
$
$ sed -i -e "/[.][${LABELS}]/  s/rw/rw,${OPTIONS}/" ${FSTAB_FILE}
$ cat /etc/fstab
092510a73bfe1df6.b none swap sw
092510a73bfe1df6.a / ffs rw,noatime,softdep 1 1
092510a73bfe1df6.k /home ffs rw,noatime,softdep,nodev,nosuid 1 2
092510a73bfe1df6.d /tmp ffs rw,noatime,softdep,nodev,nosuid 1 2
092510a73bfe1df6.f /usr ffs rw,noatime,softdep,nodev 1 2
092510a73bfe1df6.g /usr/X11R6 ffs rw,noatime,softdep,nodev 1 2
092510a73bfe1df6.h /usr/local ffs rw,noatime,softdep,wxallowed,nodev 1 2
092510a73bfe1df6.j /usr/obj ffs rw,noatime,softdep,nodev,nosuid 1 2
092510a73bfe1df6.i /usr/src ffs rw,noatime,softdep,nodev,nosuid 1 2
092510a73bfe1df6.e /var ffs rw,noatime,softdep,nodev,nosuid 1 2
```

It seems 'l' and 'm' could be removed. It would be better if I added a comment.

Reply:
I think it'd be easier to understand if you matched to all ffs filesystems and added noatime/softdep there.